segmenter/video_segmenter_test.go

```diff
@@ -0,0 +1,202 @@
+package segmenter
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"testing"
+
+	"time"
+
+	"strconv"
+
+	"io/ioutil"
+
+	"github.com/kz26/m3u8"
+	"github.com/livepeer/lpms/stream"
+	"github.com/livepeer/lpms/vidplayer"
+	"github.com/nareix/joy4/av"
+	"github.com/nareix/joy4/av/avutil"
+	"github.com/nareix/joy4/format"
+	"github.com/nareix/joy4/format/rtmp"
+)
+
+type TestStream struct{}
+
+func (s *TestStream) GetStreamID() string { return "test" }
+func (s *TestStream) Len() int64          { return 0 }
+func (s *TestStream) ReadRTMPFromStream(ctx context.Context, dst av.MuxCloser) error {
+	format.RegisterAll()
+	wd, _ := os.Getwd()
+	file, err := avutil.Open(wd + "/test.flv")
+	if err != nil {
+		fmt.Println("Error opening file: ", err)
+		return err
+	}
+	header, _ := file.Streams()
+
+	dst.WriteHeader(header)
+	for {
+		pkt, err := file.ReadPacket()
+		if err == io.EOF {
+			dst.WriteTrailer()
+			return err
+		}
+		dst.WritePacket(pkt)
+	}
+}
+func (s *TestStream) WriteRTMPToStream(ctx context.Context, src av.DemuxCloser) error { return nil }
+func (s *TestStream) WriteHLSPlaylistToStream(pl m3u8.MediaPlaylist) error            { return nil }
+func (s *TestStream) WriteHLSSegmentToStream(seg stream.HLSSegment) error             { return nil }
+func (s *TestStream) ReadHLSFromStream(buffer stream.HLSMuxer) error                  { return nil }
+
+func TestSegmenter(t *testing.T) {
+	workDir := "/Users/erictang/Code/go/src/github.com/livepeer/lpms/segmenter/tmp" // "/Users/erictang/Code/go/src/github.com/livepeer/go-livepeer/livepeer/storage/streaming/tmp"
```

Comment:
Should we make this a directory not specific to your machine so that the tests can run on CI and elsewhere?